### PR TITLE
chore(workshops): replace 'Mar High' with 'Linda Guerresi' in workshops/ section

### LINKED ISSUE #1755: Workshops/ Page is using Linda Gerrisi's photo but still showing Mar High as author
Link: https://simplabs.com/resources/

image
image

### PR DESCRIPTION
**Pull request by oscard0m**
Fixes #1755

### CODE REVIEW

File: _workshops/design-system-kickoff-interface-inventory/description.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Design system kickoff (interface inventory)
-label: Remote workshop <!--break-->with Mar High
+label: Remote workshop <!--break-->with Linda Guerrisi High
```
Comment (marcoow) [inline _workshops/design-system-kickoff-interface-inventory/description.md]:
```suggestion
label: Remote workshop <!--break-->with Linda Guerrisi
```

File: _workshops/digital-product-strategy/description.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Digital product strategy workshop
-label: 2–3 day remote workshop <!--break-->with Mar High & Marco Otte-Witte
+label:
+  2–3 day remote workshop <!--break-->with Linda Guerrisi High & Marco
```
Comment (marcoow) [inline _workshops/digital-product-strategy/description.md]:
```suggestion
  2–3 day remote workshop <!--break-->with Linda Guerrisi & Marco
```

**Review by marcoow — DISMISSED** [review]

**Comment by marcoow** [comment]
Sorry, meant to request changes, not approve – see above

File: _workshops/design-system-kickoff-interface-inventory/description.md (hunk 1 above)
Comment (oscard0m) [reply _workshops/design-system-kickoff-interface-inventory/description.md]: 🤦🏽

**Review by marcoow — APPROVED** [review]

**Pull request merged**